### PR TITLE
Rerender on `to` change

### PR DESCRIPTION
**Pull request by knownasilya**
Basically if you change the value of `to`, the wormhole should rerender.

### CODE REVIEW

**Comment by knownasilya** [comment]
ping @lukemelia @stefanpenner

**Comment by krisselden** [comment]
I have a refactor in mind for moving this away from private APIs, and allow it to be bound without rerender

**Comment by knownasilya** [comment]
@krisselden I heard from @lukemelia, but it seems like there is no time-table yet. Would love this in the mean-time.

File: addon/components/ember-wormhole.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+  
   destinationElementId: null,
+  to: computed.alias('destinationElementId'),
+  
   destinationElement: computed('destinationElementId', function() {
     return document.getElementById(this.get('destinationElementId'));
   }),
 
+  rerenderOnDestinationChange: observes('destinationElementId', function() {
+    this.rerender();
```
Comment (stefanpenner) [inline addon/components/ember-wormhole.js]: `run.once`

Comment (knownasilya) [inline addon/components/ember-wormhole.js]: Thanks.

**Comment by lukemelia** [comment]
@knownasilya the spirit of this has been implemented as part of the public API branch that was just merged. https://github.com/yapplabs/ember-wormhole/blob/master/addon/components/ember-wormhole.js#L28-L33 New release coming shortly. Thanks for bringing up this idea!

**Pull request closed without merging**